docs/source/guide/ml.md

```diff
@@ -122,6 +126,27 @@ You can prompt your model to train in several ways:
 
 In development mode, training logs appear in the web browser console. In production mode, you can find runtime logs in `my_backend/logs/uwsgi.log` and RQ training logs in `my_backend/logs/rq.log` on the server running the ML backend, which might be different from the Label Studio server. To see more detailed logs, start the ML backend server with the `--debug` option. 
 
+### Active learning approach
+
+Active learning approach is training method with model finetuning  oppose to classical approach when you train model in batches.
```

Comment:
fine tuning

Reply:
The active learning approach is a training method with model fine-tuning opposite to the classical approach when you train models in batches.